src/console_cmds.cpp

```diff
@@ -1891,7 +1891,7 @@ DEF_CONSOLE_CMD(ConContent)
 
 	if (strcasecmp(argv[1], "unselect") == 0) {
 		if (argc <= 2) {
-			IConsoleError("You must enter the id.");
+			IConsolePrint(CC_ERROR, "You must enter the id.");
```

Comment:
While at it, remove the dot at the end of the line for consistency? Similar in other cases.

Reply:
The real question with the consistency is, remove all the dots or add all the dots?

Reply:
Good question :P Personally, I think sentences should always end with a dot, so there is that. But I am fully okay with either one .. and even the 3rd hidden option: "I am not doing that" :)